WordPress/Classes/Utility/WPWebViewController.m

```diff
@@ -206,10 +206,6 @@ - (NSString *)documentTitle
 
 - (void)loadWebViewRequest
 {
-    if ([ReachabilityUtils alertIsShowing]) {
```

Comment:
This did not seem to do anything and the functionality still works fine as far as I can tell.